catalog/package.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "@codemod.com/pnpm-catalog",
```

Comment:
how do we publish to this scope? We don't own this scope.

Reply:
changed scope, we don't publish to it